golang/cosmos/x/swingset/client/cli/tx.go

```diff
@@ -113,3 +116,89 @@ func GetCmdProvisionOne() *cobra.Command {
 	flags.AddTxFlagsToCmd(cmd)
 	return cmd
 }
+
+func NewCmdSubmitCoreEvalProposal() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "swingset-core-eval [[permit.json] [code.js]]...",
+		Args:  cobra.MinimumNArgs(2),
+		Short: "Submit a proposal to evaluate code in the SwingSet core",
+		Long: `Submit a SwingSet evaluate core Compartment code proposal along with an initial deposit.
+Specify at least one pair of permit.json and code.js files`,
```

Comment:
It will be an evaluable string.  Understanding the format of it (as well as the `permit.json`) requires understanding the `@agoric/vats/src/core/` system.  There will certainly be documentation, but this help string is not the place for it.